rest_api/sawtooth_rest_api/rest_api.py

```diff
@@ -29,10 +29,10 @@ def parse_args(args):
                         default=8080)
     parser.add_argument('--host',
                         help='The host for the api to run on',
-                        default="0.0.0.0")
+                        default="127.0.0.1")
     parser.add_argument('--stream-url',
                         help='The url to connect to a running Validator',
-                        default='tcp://localhost:40000')
+                        default='tcp://127.0.0.1:40000')
```

Comment:
This change is not required. While the bind() needs an interface definition, the connect() can take a dns name. I think the dns name is clearer.